core/store/src/trie/state_snapshot.rs

```diff
@@ -7,14 +7,63 @@ use crate::{Store, StoreConfig};
 use near_primitives::block::Block;
 use near_primitives::errors::EpochError;
 use near_primitives::errors::StorageError;
-use near_primitives::errors::StorageError::StorageInconsistentState;
 use near_primitives::hash::CryptoHash;
 use near_primitives::shard_layout::ShardUId;
 
+use std::error::Error;
 use std::io;
 use std::path::{Path, PathBuf};
 use std::sync::TryLockError;
 
+#[derive(Debug, PartialEq, Eq)]
+pub enum SnapshotError {
+    // The requested hash and the snapshot hash don't match.
```

Comment:
nit: I think triple / is better because then it becomes a 'doc'. I think IDE maybe picks up on it? Not entirely sure.